sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/EndpointConfig.java

```diff
@@ -4,185 +4,206 @@
-			this.cassandraiP = cassandraiP;
-		}
-
-		public String getCassandraPort() {
-			return cassandraPort;
-		}
-
-		public void setCassandraPort(String cassandraPort) {
-			this.cassandraPort = cassandraPort;
-		}
-
-		public String getKeySpace() {
-			return keySpace;
-		}
-
-		public void setKeySpace(String keySpace) {
-			this.keySpace = keySpace;
-		}
-
-		public String getCassandraUserName() {
-			return cassandraUserName;
-		}
-
-		public void setCassandraUserName(String cassandraUserName) {
-			this.cassandraUserName = cassandraUserName;
-		}
-
-		public String getClientId() {
-			return clientId;
-		}
-
-		public void setClientId(String clientId) {
-			this.clientId = clientId;
-		}
-
-		public String getEsHost() {
-			return esHost;
-		}
-
-		public void setEsHost(String esHost) {
-			this.esHost = esHost;
-		}
-
-		public String getEsPort() {
-			return esPort;
-		}
-
-		public void setEsPort(String esPort) {
-			this.esPort = esPort;
-		}
-
-		public String getIndex() {
-			return index;
-		}
-
-		public void setIndex(String index) {
-			this.index = index;
-		}
-
-		public String getIndexType() {
-			return indexType;
-		}
-
-		public void setIndexType(String indexType) {
-			this.indexType = indexType;
-		}
-
-		@Override
-		public String toString() {
-			return "TestGlobalProperty [apiKey=" + apiKey + ", keycloakAdminUser=" + keycloakAdminUser
-					+ ", keycloakAdminPass=" + keycloakAdminPass + ", relam=" + relam + ", cassandraiP=" + cassandraiP
-					+ ", cassandraPort=" + cassandraPort + ", keySpace=" + keySpace + ", cassandraUserName="
-					+ cassandraUserName + ", clientId=" + clientId + "]";
-		}
-
-	}
-
-	public static String val;
-
-	static {
-		val = UUID.randomUUID().toString();
-	}
+  @Bean
+  public HttpClient restTestClient() {
+    return CitrusEndpoints.http().client().requestUrl(System.getenv("test_base_url")).build();
+  }
+
+  @Bean
+  public TestGlobalProperty initGlobalValues() {
+    TestGlobalProperty property = new TestGlobalProperty();
+    property.setApiKey(System.getenv("sunbird_api_key"));
+    property.setCassandraiP(System.getenv("sunbird_cassandra_host"));
+    property.setCassandraPort(System.getenv("sunbird_cassandra_port"));
+    property.setCassandraUserName(System.getenv("sunbird_cassandra_username"));
+    property.setKeySpace(System.getenv("sunbird_cassandra_keyspace"));
+    property.setKeycloakAdminUser(System.getenv("sunbird_sso_username"));
+    property.setKeycloakAdminPass(System.getenv("sunbird_sso_password"));
+    property.setRelam(System.getenv("sunbird_sso_realm"));
+    property.setClientId(System.getenv("sunbird_sso_client_id"));
+    property.setEsHost(System.getenv("sunbird_es_host"));
+    property.setEsPort(System.getenv("sunbird_es_port"));
+    property.setIndexType(System.getenv("sunbird_es_index_type"));
+    property.setIndex(System.getenv("sunbird_es_index"));
+    property.setSunbirdDefaultchannel(System.getenv("sunbird_default_channel"));
```

Comment:
make sunbirdDefaultchannel to sunbirdDefaultChannel (it's three different words)